### ISSUE TITLE
疑问

### ISSUE THREAD

**Issue by olwsx** [xuanbo/fine-report#1]
和官方的有什么不同？所有功能都没限制吗？有移动端的功能吗

**Comment by xuanbo** [comment]
跟官方的一样，只是说将原来的tomcat部署方式，改为spring boot fatjar启动。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"olwsx"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月31日(星期二) 上午9:11
收件人:&nbsp;"xuanbo/fine-report"<fine-report@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[xuanbo/fine-report] 疑问 (#1)




和官方的有什么不同？所有功能都没限制吗？有移动端的功能吗

**Comment by olwsx** [comment]
那也要license的吧？

**Comment by xuanbo** [comment]
是的。我这个只是将帆软传统的war部署，改成spring boot项目，方便升级为spring cloud微服务，后面网关加入hash一致性路由，将其做成帆软报表集群。
